### PR TITLE
feat: Add lunch map

### PR DESCRIPTION
**Pull request by koji-1009**
## Description

* Add lunch map
* Added remote config key for lunch stores, and add json parsing process

Refs #20

## Type of change

- [x] New feature

## How Has This Been Tested?

Local build.

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] My code has been formatted with `flutter format lib`.
- [x] My code has been fixed with `dart fix --apply lib`.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: lib/ui/screen/venue/lunch_map.dart [hunk 1]
```diff
@@ -1,14 +1,156 @@
...
+    final storeList = ref.watch(storeListProvider);
+
+    return TableView.builder(
+      pinnedRowCount: 1,
+      columnCount: 7,
+      rowCount: storeList.items.length,
+      columnBuilder: (index) => switch (index) {
+        0 => const TableSpan(
+            extent: FixedTableSpanExtent(60),
+          ),
```
Comment (Kurogoma4D) [inline lib/ui/screen/venue/lunch_map.dart:24]: 他の箇所はまだいいかもしれませんが、モバイルで見ると結構番号の幅が広く見えるのでもうちょっと詰められそうです 🧐 
40〜48くらいで良さそうな気がします。

File: lib/ui/screen/venue/venue.dart [hunk 2]
```diff
@@ -65,8 +65,7 @@ class VenuePage extends ConsumerWidget {
             VenueTab.floor => const FloorMapPage(),
             VenueTab.location =>
               _isProduction ? const LocationMapPage() : const VenueDevDummy(),
-            VenueTab.lunch =>
-              _isProduction ? const LaunchMapPage() : const VenueDevDummy(),
+            VenueTab.lunch => const LaunchMapPage(),
```
Comment (Kurogoma4D) [inline lib/ui/screen/venue/venue.dart:68]: そういえば `Launch` になってるので `Lunch` に修正お願いします（SegmentedButtonのラベルの方も） 🙏

File: lib/ui/screen/venue/lunch_map.dart (hunk 1 above)
Comment (koji-1009) [reply lib/ui/screen/venue/lunch_map.dart]: [9e609a5](https://github.com/FlutterKaigi/conference-app-2023/pull/182/commits/9e609a5cc30bb790978641bd6e40432256d4f8c8) で修正してみました。TableViewにどんな感じでpaddingを与えるといいのか、全然わからない。。

**Review by Kurogoma4D — APPROVED** [review]
LGTM~

**Pull request merged**